app/models/manageiq/providers/hawkular/middleware_manager.rb

```diff
@@ -12,6 +12,7 @@ class Hawkular::MiddlewareManager < ManageIQ::Providers::MiddlewareManager
     require_nested :MiddlewareDatasource
     require_nested :MiddlewareMessaging
     require_nested :MiddlewareServer
+    require_nested :RefreshWorker
```

Comment:
I will follow your suggestion and rearrange the commits :)
Thank you very much!